.htaccess

```diff
@@ -67,6 +67,15 @@ deny from env=stayout
 	RewriteRule src/Frontend/Cache/Search - [F]
 	RewriteRule src/Backend/Cache/CompiledTemplates - [F]
 
+	# @remark: custom for Sumocoders
+	RewriteRule .*\.rb - [F]
+	RewriteRule Capfile - [F]
+	RewriteRule Gemfile - [F]
+	RewriteRule Gemfile\.lock - [F]
+	RewriteRule Gruntfile\.coffee - [F]
+	RewriteRule Sumofile - [F]
+	RewriteRule package\.json - [F]
```

Comment:
this is also present in fork, so this is not a custom one but needs to be added to fork and then pulled in here